RickLeinecker/Fortuna
# Search Players functionality.

**Description**
Added API functionality for Search Players.

**Type of change**
Check options that are relevant.


**Steps to Verify Functionality**
1. Ensure users are on the backend.
2. Log in and go to Battle Arena.
3. Type a user's name in the searchbox or press enter on blank to see everyone.

**Final Checklist:**
Go down the list and check them off.
